samples/03_org_administrators/clone_a_group.ipynb

```diff
@@ -236,34 +170,34 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 10,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/html": [
        "<div class=\"9item_container\" style=\"height: auto; overflow: hidden; border: 1px solid #cfcfcf; border-radius: 2px; background: #f6fafa; line-height: 1.21429em; padding: 10px;\">\n",
        "                    <div class=\"item_left\" style=\"width: 210px; float: left;\">\n",
-       "                       <a href='https://pythonapi.playground.esri.com/portal/home/group.html?id=32d87e01dd734563a377a1c5b82425fa' target='_blank'>\n",
+       "                       <a href='https://rpubw120iwapt.ags.esri.com/portal/home/group.html?id=84664d956490472f89d7ba14a84fbed8' target='_blank'>\n",
```

Comment:
@jyaistMap Unsure about this one so will defer to you but do we want to expose this enterprise?